.github/workflows/build_and_test.yml

```diff
@@ -319,3 +319,29 @@ jobs:
         ./dev/change-scala-version.sh 2.13
         ./build/mvn $MAVEN_CLI_OPTS -DskipTests -Pyarn -Pmesos -Pkubernetes -Phive -Phive-thriftserver -Phadoop-cloud -Djava.version=11 -Pscala-2.13 install
         rm -rf ~/.m2/repository/org/apache/spark
+
+  scala-213-sbt:
```

Comment:
@sarutak, can we actually fix the main job https://github.com/apache/spark/blob/5dec1ffc9eaae07b6c71ebb2de4b64da87328390/.github/workflows/build_and_test.yml#L16-L80 to run the tests?